src/renderers/server-render.jsx

```diff
@@ -94,7 +93,10 @@ const getRouterRenderer = (store, baseUrl, clientFilename, assetPublicPath) =>
 				200;
 		} catch(e) {
 			// log the error stack here because Observable logs not great
-			console.error(e.stack);
+			console.error(JSON.stringify({
+				message: `Internal rendering error ${e.message}`,
```

Comment:
I'm not entirely sure what's in `e.message` and `e.stack`, but I assume that it doesn't have the URL that the user was hitting. That might be nice info to pass along?